python/demo/nessie-delta-demo.ipynb

```diff
@@ -31,7 +31,7 @@
     "                    .config(\"spark.hadoop.fs.defaultFS\", 'file://' + os.getcwd() + '/spark_warehouse') \\\n",
     "                    .config(\"spark.hadoop.nessie.uri\", \"http://localhost:19120/api/v1\") \\\n",
     "                    .config(\"spark.hadoop.nessie.ref\", \"main\") \\\n",
-    "                    .config(\"spark.hadoop.nessie.auth_type\", \"NONE\") \\\n",
+    "                    .config(\"spark.hadoop.nessie.auth-type\", \"NONE\") \\\n",
```

Comment:
Do we not have to make any code changes to match this renaming? :thinking:

Reply:
Those were already done a while ago but we were waiting for Iceberg to be released to update the other places here